### DISCUSSION TITLE
Allow queries to be fuzzy

### DISCUSSION THREAD

**Discussion by tlammer** [specify/specify7#5547] [Feature Requests]
Users requested being able to do fuzzy queries.
From SDSMT, Nate Fox
To follow up on my earlier request, it would be nice if the locality number query was more robust to formatting errors in future updates. For example, our locality numbers are organized by year-number. They have been written both with and without a dash between the year and number in the past (e.g., V93-15, V9315). I've noticed that Specify queries using the "contains" operator often miss records depending on whether the dash is entered or not. This formatting/query issue extends to other fields as well (e.g., Field Number X-XX-XXX versus XXX-XXX). Modifying queries to be less sensitive to minor formatting errors, especially dashes and spaces, would be a huge help for us.

**Comment by maxpatiiuk** [comment]
As a short-term workarround, "Like" filter supports advanced features that can cover many use cases:

<img width="1397" alt="Screenshot 2023-03-07 at 21 42 01" src="https://user-images.githubusercontent.com/40512816/223614000-b70734d9-32e5-400c-b123-66afa6f46266.png">

(from https://www.w3schools.com/sql/sql_like.asp)
